scripts/run_tracker.py

```diff
@@ -172,6 +177,154 @@ def trigger_analysis(run, config):
             LOG.warn(("Something went wrong when triggering the analysis of {}. Please "
                       "check the logfile and make sure to start the analysis!".format(os.path.basename(run))))
 
+def prepare_sample_sheet(run, config):
+    """ This is a temporary function in order to solve the current problem with LIMS system
+        not able to generate a compatible samplesheet for HiSeqX. This function needs to massage 
+        the sample sheet created by GenoLogics in order to correctly demultiplex HiSeqX runs.
+        This function returns with success if the samplesheet is in the correct place, otherwise
+        this flowcell will not be processed.
+        
+        :param str run: Run directory
+        :param dict config: Parset configuration file
+        
+    """
+    #start by checking if samplesheet is in the correct place
+    run_name     = os.path.basename(run)
+    current_year = '20' + run_name[0:2]
```

Comment:
@guillermo-carrasco no it might fail, if the flowcell is created the 31st of December and demultiplexed the 3rd of January checking the current date will cause problems.